zebedee-reader/src/test/java/com/github/onsdigital/zebedee/reader/api/bean/PublishedIndexResponseTest.java

```diff
@@ -0,0 +1,101 @@
+package com.github.onsdigital.zebedee.reader.api.bean;
+
+import com.github.onsdigital.zebedee.search.indexing.Document;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.*;
```

Comment:
Can you update your ide setting to prevent wildcard imports

https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports/25916603